SLHCUpgradeSimulations/Configuration/python/gemCustom.py

```diff
@@ -9,25 +9,24 @@ def customise2023(process):
   return process
 def customise_digitization(process):
   from SimMuon.GEMDigitizer.customizeGEMDigi import customize_digi_addGEM_muon_only
-  process = customize_digi_addGEM_muon_only(process)
-  process.simMuonGEMDigis.mixLabel = cms.string("mix")
-  process.simMuonME0Digis.mixLabel = cms.string("mix")
-  # process.digitisation_step.remove(process.simMuonRPCDigis)
-  # process.simMuonRPCDigis.digiModel = cms.string('RPCSimParam')
-  process.simMuonRPCDigis.digiModel = cms.string('RPCSimAverageNoiseEff')
+  if ( not hasattr(process,"simMuonGEMDigis")) :
+    process = customize_digi_addGEM_muon_only(process)
+  if ( not hasattr(process,"simMuonGEMDigis.mixLabel")) :
```

Comment:
@geonmo - I notice only now - this is not proper python. you need to test first hasattr(process,'simMuonGEMDigis') and then hasattr(process.simMuonGEMDigis,'mixLablel')